src/applications/static-pages/health-care-manage-benefits/get-medical-records-page/components/AuthContent/index.js

```diff
@@ -142,6 +142,12 @@ const AuthContent = ({
         </a>
       </li>
     </ul>
+    <a
+      href="https://www.myhealth.va.gov/mhv-portal-web/my-healthevet-offers-three-account-types"
+      rel="noreferrer noopener"
```

Comment:
Do we want `noreferrer` if we're linking to a va.gov domain?

Reply:
Any external domain I was told 🤷